docs/apis/common/index.md

```diff
@@ -0,0 +1,1356 @@
+---
+title: "Basic Concepts"
+
+# Top-level navigation
+top-nav-group: apis
+top-nav-pos: 1
+top-nav-title: <strong>Basic Concepts</strong>
+
+# Sub-level navigation
+sub-nav-group: common
```

Comment:
Maybe we can comment these lines out, because this is currently just a single page. It shows a navigation at the left side with only one element. Or do you think that we will add sub pages soon?